internal/logger/target/http/http.go

```diff
@@ -219,17 +221,23 @@ func (h *Target) initMemoryStore(ctx context.Context) (err error) {
 	return nil
 }
 
-func (h *Target) send(ctx context.Context, payload []byte, payloadType string, timeout time.Duration) (err error) {
+func (h *Target) send(ctx context.Context, payload []byte, payloadCount int, payloadType string, timeout time.Duration) (err error) {
+	h.failedMessages.Add(int64(payloadCount))
```

Comment:
won't this produce spikes in `minio_audit_failed_messages` metric even when there aren't any real failed messages?

Reply:
don't you see we remove this when we are returning? @Praveenrajmani

Reply:
look at `if err == nil` in the defer()

Reply:
yeah, if send() takes more time to complete and stats() is called in between, there will be spikes which are not really failures right?

Reply:
it is @Praveenrajmani if we can't send() it is a failure, doesn't matter if we might send later.

Reply:
this is a counter, it is not an exact value of sent() events.